numpy/fft/_pocketfft.py

```diff
@@ -67,10 +67,12 @@ def _raw_fft(a, n, axis, is_real, is_forward, inv_norm):
             a = z
 
     if axis == a.ndim-1:
-        r = pfi.execute(a, is_real, is_forward, fct)
+        r = pfi.execute(a, is_real, is_forward, fct, out)
     else:
         a = swapaxes(a, axis, -1)
-        r = pfi.execute(a, is_real, is_forward, fct)
+        if out is not None:
+            out = swapaxes(a, axis, -1)
```

Comment:
This will generally make `out` not C contiguous. Maybe add a test for that too. Though that is it so sensitive to that worries me...